SplitFiles/slot_monitor.c

```diff
@@ -101,9 +102,23 @@ SlotMonitor* get_slot_monitor(unsigned char x, unsigned char y, int slotIndex) {
 
     //Set up attributes - Blue x2, Magenta x2, Red x2, Black x2 - all with white paper
     //const unsigned char* attrs = (unsigned char*)"\x79\x79\x7B\x7B\x7A\x7A\x78\x78"; //BRIGHT
-    const unsigned char* attrs = (unsigned char*)"\x39\x39\x3B\x3B\x3A\x3A\x38\x38"; //DIMMER
-    memcpy(slotMon->startAttributes, attrs, 7);
-    memcpy(slotMon->startAttributes + 32, attrs, 7); //Next row down
+    //const unsigned char* attrs = (unsigned char*)"\x39\x39\x3B\x3B\x3A\x3A\x38\x38"; //DIMMER
```

Comment:
Can't help thinking that 8 function calls is going to have a significant overhead beyond a single memcpy call?

We're pretty resource-constrained here - as an example compile music.c on its own (command is in the code) and run it.  The size of the text in the print statement significantly slows down the music playback as the number of digits increases.

I'm probably inclined towards writing as much "bare metal" code as we can handle on the assumption it'll be faster.  That may be a false assumption of course,

Reply:
My feeling is that when I replace your other code it will be fewer lines there, after all no need for the offset calculations etc. Let me try that and see what you think.

Reply:
More interested in overall speed of the code than line count - I'm happy for the code to be verbose if performance is good.  This particular case is low impact (done once at set-up) but the drawing of the bars might be a bottleneck if the trade-off looks similar there